Sources/StreamChatUI/CommonViews/Suggestions/ChatSuggestionsVC.swift

```diff
@@ -189,6 +189,9 @@ open class ChatMessageComposerSuggestionsCommandDataSource: NSObject, UICollecti
 open class ChatMessageComposerSuggestionsMentionDataSource: NSObject,
     UICollectionViewDataSource,
     ChatUserSearchControllerDelegate {
+    /// internal cache for users
+    private(set) var usersCache: [ChatUser]
```

Comment:
Small detail: I think naming this just `users` is enough?
```suggestion
    private(set) var users: [ChatUser]
```